zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/changes/PartitionEnableExporterApplier.java

```diff
@@ -95,6 +95,14 @@ public Either<Exception, UnaryOperator<MemberState>> initMemberState(
 
     final var partitionHasExporter = exportersInPartition.containsKey(exporterId);
 
+    if (partitionHasExporter && exportersInPartition.get(exporterId).state() == State.ENABLED) {
+      return Either.left(
+          new IllegalStateException(
+              String.format(
+                  "Expected to enable exporter, but the exporter '%s' is already enabled",
+                  exporterId)));
+    }
+
```

Comment:
:thought_balloon: What should be the behavior if user try to enable and already enabled exporter? I think it is better to reject it, otherwise the semantics is not clear.

Reply:
I guess it depends practically how it will be used. If we want to allow idempotent retries, then it makes sense to make it a noop. Otherwise, we should reject it with a clear and explicit errors so users can decide whether or not to retry, possibly in an automated way.

Reply:
I made retry a noop in ExporterDirector and removed this short circuit here.

Reply:
I will leave it as it is. That is retry will fail. There is not much use in retry anyway, as the monitoring API should give enough info for the user to determine if they should retry or not. We can allow idempotent retry later if needed.